### ISSUE TITLE
Uvula papilloma

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6852]
**Preferred term label:**
Uvula papilloma

**Synonyms**
Papilloma of the uvula

**Definition (free text, please give PubMed ID)**
A papilloma located in the uvula. 
PMID: 32133247

**Parent term (use hpo.jax.org/app)**
Abnormality of the uvula

**Comment by pnrobinson** [comment]
This type of neoplasm is actually Oropharyngeal squamous papilloma
https://hpo.jax.org/app/browse/term/HP:0031022
It does not make sense to have a subterm for each location in the oral cavity. I am therefore going to add squamous papilloma of the uvula as a narrow synonym.
